components/concept/engine/src/main/java/mozilla/components/concept/engine/activity/ActivityOrientationDelegate.kt

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.activity
+
+import android.content.pm.ActivityInfo
+
+/**
+ * Notifies applications or other components of engine orientation lock events.
+ */
+interface ActivityOrientationDelegate {
```

Comment:
```suggestion
interface OrientationDelegate {
```

Does this need to be an "Activity" delegate? For WebAuthn, we added an explicit `ActivityDelegate` because that was a 1-1 mapping with what GeckoView needed for that feature. Whereas for the orientation delegate, I debated whether this needs to have that naming. 

@calumozilla and I spoke about this and we figured it was fine since the `ActivityInfo` values are a hard requirement, but I thought I'd share our discussion here as well.

Reply:
I used `Activity` here because implementation would set the orientation using using an Activity. But this would be an implementation detail not necessarily known by the interface.
Will remove it, thanks!